pages/dkp/kaptain/2.0.0/install/konvoy-dkp/index.md

```diff
@@ -36,72 +40,57 @@ The amounts depend on the number, complexity, and size of the workloads, as well
-    configVersion: stable-1.20-4.1.0
-    addonsList:
-      - name: istio
-        enabled: true
-      - name: dex
-        enabled: true
-      - name: cert-manager
-        enabled: true
-      - name: prometheus
-        enabled: true
-  ```
+## Prerequisites for DKP 2.x
 
-- Add the Kaptain addon repository to your Konvoy `cluster.yaml` to install Kaptain dependencies:
-  ```yaml
-  - configRepository: https://github.com/mesosphere/kubeaddons-kaptain
-    configVersion: stable-1.20-1.4.0
-    addonsList:
-      - name: knative
-        enabled: true
-  ```
-- For GPU deployment, follow the instructions in [Konvoy GPU documentation][konvoy-gpu].
-- Then follow the [Konvoy documentation][konvoy_deploy_addons] to deploy the addons.
+-   A DKP cluster with the following Platform applications enabled:
 
-## Prerequisites for DKP 2.x
+    - Istio
+    - Knative
 
-For DKP 2.x, ensure the following applications are enabled in Kommander:
+-   [`kubectl`][kubectl] on your installation machine
 
-- Use the existing Kommander configuration file, or initialize the default one:
-  ```
-  kommander install --init > kommander-config.yaml
-  ```
-- Ensure the following applications are enabled in the config:
-  ```yaml
-  apiVersion: config.kommander.mesosphere.io/v1alpha1
-  kind: Installation
-  apps:
-    ...
-    dex:
-    dex-k8s-authenticator:
-    kube-prometheus-stack:
-    istio:
-    knative:
-    minio-operator:
-    traefik:
-    nvidia:  # to enable GPU support
-    ...
-  ```
-- For GPU deployment, follow the instructions in [Kommander GPU documentation][kommander-gpu].
+-   For multi-cluster environments (Enterprise license): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex]. 
 
-- Apply the new configuration to Kommander:
-  ```
-  kommander install --installer-config kommander-config.yaml
-  ```
-  Check [Kommander installation documentation][kommander-install] for more information.
+-   For DKP 2.x, ensure the following applications are enabled in Kommander:
+
+    1. Use the existing Kommander configuration file, or initialize the default one:
+
+    ```bash
+    kommander install --init > kommander-config.yaml
+    ```
+
+    1. Ensure the following applications are enabled in the config:
+
+    ```yaml
+    apiVersion: config.kommander.mesosphere.io/v1alpha1
+    kind: Installation
+    apps:
+      ...
+      dex:
+      dex-k8s-authenticator:
+      kube-prometheus-stack:
+      istio:
+      knative:
+      minio-operator:
+      traefik:
+      nvidia:  # to enable GPU support
+      ...
+    ```
+
+    1. For GPU deployment, follow the instructions in [Kommander GPU documentation][kommander-gpu].
+
+    1. Apply the new configuration to Kommander:
+    
+    ```bash
+    kommander install --installer-config kommander-config.yaml
+    ```
+
+    Check [Kommander installation documentation][kommander-install] for more information.
```

Comment:
Can we delete lines 54 to 88? Or are these steps necessary? @lukeogg @alembiewski